pkgs/tools/backup/mackup/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, fetchurl
+, python2Packages
+, stdenv
+}:
+
+python2Packages.buildPythonApplication rec {
+    name = "mackup-0.8.16";
+
+    src = fetchurl {
+        url = "https://pypi.python.org/packages/e5/3a/e8a422e88dab6f734706a7612adc4715cc44512d48845295ceea7a9fc244/${name}.tar.gz";
+        sha256 = "0f5cb5srxa3bn7lbfs5rb4iqjs57blzvgypivndy756vsvm9ijrv";
+    };
+
+    "docopt" = python2Packages.buildPythonPackage {
```

Comment:
Please put this in `pkgs/development/python-modules` and use `callPackage` in `python-packages.nix`.